src/ceph-disk/ceph_disk/main.py

```diff
@@ -3001,10 +3001,19 @@ def start_daemon(
                 ],
             )
         elif os.path.exists(os.path.join(path, 'systemd')):
+            # ensure there is no duplicate ceph-osd@.service
+            command_check_call(
+                [
+                    'systemctl',
+                    'disable',
+                    'ceph-osd@{osd_id}'.format(osd_id=osd_id),
```

Comment:
of course, it's not likely to happen and that kind of change falls into the "cleanup" category.